third_party/blink/renderer/core/brave_page_graph/blink_probe_types.h

```diff
@@ -6,14 +6,12 @@
 #ifndef BRAVE_THIRD_PARTY_BLINK_RENDERER_CORE_BRAVE_PAGE_GRAPH_BLINK_PROBE_TYPES_H_
 #define BRAVE_THIRD_PARTY_BLINK_RENDERER_CORE_BRAVE_PAGE_GRAPH_BLINK_PROBE_TYPES_H_
 
-#include <vector>
-
 #include "third_party/blink/renderer/platform/wtf/hash_map.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
 namespace blink {
 
-using PageGraphBlinkArgs = std::vector<String>;
+using PageGraphBlinkArgs = WTF::Vector<WTF::String>;
```

Comment:
`Vector<String>`
because blink has `using WTF::Vector;` and `using WTF::String;`.

Reply:
Done